src/app/cases-overview/cases-overview.component.ts

```diff
@@ -0,0 +1,64 @@
+import { SelectionModel } from '@angular/cdk/collections';
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Sort } from '@angular/material/sort';
+import { CaseDataDto } from 'api-client';
+import { Subscription } from 'rxjs';
+import { CaseService } from '../services/case.service';
+import { PaginatedDataService } from '../services/paginated-data.service';
+import { ApiError } from '../shared/http/BaseDataService';
+import { TableColumn } from '../shared/table/table-column';
+import { defaultColumnDefs } from './columns-default';
+
+const VISIBLE_ROWS_COUNT = 8;
+const PAGE_SIZE = 8;
```

Comment:
we should make a constants file and add these in the constants, because we will use them in a lot of files